Source/FuseJS/Bundle.uno

```diff
@@ -53,14 +53,15 @@ namespace FuseJS
 			}
 			```
 		*/
-		static Future<byte[]> GetBuffer(object[] args)
+		public static Future<byte[]> ReadBuffer(object[] args)
```

Comment:
It's a general rule we're following: everything is internal/private unless we have an actual use-case that requires it. Every `public` member increase our public API coverage and limits how we can refactor, and increases what we need to support.  This PR is about a jsAPI and thus having public Uno methods is not part of the use-case.